pettingzoo/classic/chess/chess.py

```diff
@@ -53,7 +53,6 @@ def __init__(self):
         }
 
         self.rewards = None
-        self.dones = None
```

Comment:
Is there a reason this was removed but not replaced with terminations and truncations?